src/components/Button.astro

```diff
@@ -0,0 +1,178 @@
+---
+const { class: className = '', style, href } = Astro.props;
+// Wrap in <span> because Houdini is disabled for a[href] for security
+
+const { variant = 'primary' } = Astro.props;
+const { before, after } = Astro.slots;
+---
+
+<!-- Blocking script -->
+<script hoist>
```

Comment:
I think that this `/pixel.worklet.js` URL 404s, okay to remove the script entirely for now